hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/ha/MockSCMHAManager.java

```diff
@@ -71,6 +76,11 @@ private MockSCMHAManager(boolean isLeader, DBTransactionBuffer buffer) {
     this.transactionBuffer = buffer;
   }
 
+  private MockSCMHAManager(SCMDBTransactionBufferImpl buffer) {
+    this.transactionBuffer = buffer;
```

Comment:
The change in `MockSCMHAManager` is not needed. 
The bypass ratis scm is based on the `MockRatisServer`.